docs/install-understack-ubuntu-k3s.md

```diff
@@ -0,0 +1,305 @@
+# Installing UnderStack on Ubuntu 22.04 + K3s
+
+## Install Pre-requisites
+
+Install some packages we'll need later and some useful troubleshooting utilities.
+```bash
+apt-get -y install curl jq yq net-tools telnet git apt-transport-https golang-go
+```
+
+## Update Ubuntu
+
+Update to the latest ubuntu packages and reboot if necessary.
+```bash
+apt-get -y update
+```
+
+## Install K3s
+
+```bash
+curl -sfL https://get.k3s.io | sh -
+```
+
+References:
+
+* [https://docs.k3s.io/](https://docs.k3s.io/)
+
+## Install Helm
+
+The K3s installer will install kubectl, but we'll also need helm for the UnderStack install.
+
+```bash
+curl https://baltocdn.com/helm/signing.asc | gpg --dearmor | sudo tee /usr/share/keyrings/helm.gpg > /dev/null
+sudo apt-get install apt-transport-https --yes
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/helm.gpg] https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+sudo apt-get update
+sudo apt-get install helm
```

Comment:
Is this the official helm repo?

Reply:
Yes, I copy/pasted it from: https://helm.sh/docs/intro/install/#from-apt-debianubuntu